scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1691,9 +1691,7 @@ def get_user_agent_fragment(self):
         ver = None
         if k8s_cache:
             ver = k8s_cache.get_api_server_version()
-            if ver:
-                ver = 'k8s=%s' % ver
-        return ver
+        return 'k8s=%s' % (ver if ver else 'true')
```

Comment:
I owe you guys a unit test for this one (or at least manual testing before Friday release)